core/server/api_container/server/startosis_engine/startosis_validator/docker_images_validator.go

```diff
@@ -43,13 +45,13 @@ func (validator *DockerImagesValidator) Validate(ctx context.Context, environmen
 	wg := &sync.WaitGroup{}
 	for image := range environment.requiredDockerImages {
 		wg.Add(1)
-		go fetchImageFromBackend(ctx, wg, imageCurrentlyDownloading, validator.kurtosisBackend, image, pullErrors, imageDownloadStarted, imageDownloadFinished)
+		go fetchImageFromBackend(ctx, wg, imageCurrentlyDownloading, validator.kurtosisBackend, image, environment.imageDownloadMode, pullErrors, imageDownloadStarted, imageDownloadFinished)
 	}
 	wg.Wait()
 	logrus.Debug("All image validation submitted, currently in progress.")
 }
 
-func fetchImageFromBackend(ctx context.Context, wg *sync.WaitGroup, imageCurrentlyDownloading chan bool, backend *backend_interface.KurtosisBackend, imageName string, pullErrors chan<- error, imageDownloadStarted chan<- string, imageDownloadFinished chan<- *ValidatedImage) {
+func fetchImageFromBackend(ctx context.Context, wg *sync.WaitGroup, imageCurrentlyDownloading chan bool, backend *backend_interface.KurtosisBackend, imageName string, image_download_mode image_download_mode.ImageDownloadMode, pullErrors chan<- error, imageDownloadStarted chan<- string, imageDownloadFinished chan<- *ValidatedImage) {
```

Comment:
golang does camelCase as opposed to snake_case, thus `s/image_download_mode/imageDownloadMode/`

Reply:
thanks for clarifying it, I found it a bit confusing (even in golang itself is confused about it: https://github.com/golang/go/issues/36060). For instance, filename are snake_case and vars/func are CamelCase but protobuf definition has CamelCase for message names but snake_case for fields which generates :p

Reply:
oh, it would be nice to if we can find a linter that could help to enforce it.